docker/Dockerfile

```diff
@@ -92,8 +109,9 @@ RUN apt update \
   && rm -rf /var/lib/apt/lists/* \
   && rm -rf /var/cache/apt/archives/*
 
-ENV LANG   en_US.UTF-8
-ENV LC_ALL en_US.UTF-8
+ENV LANG="en_US.UTF-8" \
+  LC_ALL="en_US.UTF-8" \
+  PIP_DISABLE_PIP_VERSION_CHECK=1
```

Comment:
I've not seen Dockerfile env variables split across lines like this... I'm assuming you've tested it?

Reply:
Yes, these changes were tested, and I use multi-line ENV split in a number of other projects. Essentially it bundles all the variables into single image layer, instead of a separate layer for each variable.